File: src/main/java/org/radarbase/appserver/dto/protocol/AssessmentType.java
package org.radarbase.appserver.dto.protocol;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum AssessmentType {
    @JsonProperty("scheduled")
    SCHEDULED,
    @JsonProperty("clinical")
    CLINICAL,
    @JsonProperty("triggered")
    TRIGGERED,
    @JsonProperty("all")
    ALL
}

File: appserver-jersey/src/main/kotlin/org/radarbase/appserver/jersey/dto/protocol/AssessmentType.kt
package org.radarbase.appserver.jersey.dto.protocol

import com.fasterxml.jackson.annotation.JsonProperty

enum class AssessmentType {
    @JsonProperty("scheduled")
    SCHEDULED,

    @JsonProperty("clinical")
    CLINICAL,

    @JsonProperty("triggered")
    TRIGGERED,

    @JsonProperty("all")
    ALL,
}
